src/components/MyCalculator.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import Button from './Button';
+
+function MyCalculator() {
```

Comment:
- [ ] According to the instructions, the name of this component should be `Calculator.js` as shown in the requirements below, please let's try to follow project requirements as it is best practice for software developers.
<img width="484" alt="Screenshot 2023-07-06 at 14 51 31" src="https://github.com/yin-ka/math-magician/assets/64774936/c2884954-4381-40d0-9661-56a6e3e75381">